webpack/scenes/Subscriptions/Manifest/CdnConfigurationForm.js

```diff
@@ -130,7 +136,27 @@ const CdnConfigurationForm = (props) => {
             aria-label="cdn-organization-label"
             type="text"
             value={organizationLabel || ''}
-            onChange={value => setOrganizationLabel(value)}
+            onChange={setOrganizationLabel}
+          />
+        </FormGroup>
+        <FormGroup
+          label={__('Content View Label')}
+        >
+          <TextInput
+            aria-label="cdn-content-view-label"
+            type="text"
+            value={contentViewLabel || ''}
+            onChange={setContentViewLabel}
+          />
+        </FormGroup>
+        <FormGroup
+          label={__('Lifecycle Environment Label')}
+        >
+          <TextInput
+            aria-label="cdn-lifecycle-environment-label"
+            type="text"
+            value={lifecycleEnvironmentLabel || ''}
+            onChange={setLifecycleEnvironmentLabel}
```

Comment:
I'd probably put Lifecycle Environment first, and content view 2nd.

Its also not obvious what i'm supposed to do if i just want to use 'Library'.  Any thought on how to make that clearer?

Reply:
and i get that i just leave those two fields blank, as they aren't required, i don't think a user will figure that out immediately thought.

Reply:
![RkLMy2O](https://user-images.githubusercontent.com/1069779/149192110-c8186aa8-845d-4c49-bff5-92918ff7a9bd.png)

Added tool tips to make it clear. Will address it better in PR https://github.com/Katello/katello/pull/9812